bot/exts/info/tags.py

```diff
@@ -91,7 +91,7 @@ def embed(self) -> Embed:
         embed.description = self.content
         return embed
 
-    def accessible_by(self, member: discord.Member) -> bool:
+    def accessible_by(self, member: Member | User) -> bool:
```

Comment:
I think this can be kept to `Member`
Since the invoker needs to be a member of our server if they want to execute the command, so it won't resolve to an instance of `User`